edb/pgsql/common.py

```diff
@@ -188,17 +188,19 @@ def get_scalar_backend_name(id, module_id, catenate=True, *, aspect=None):
 def get_objtype_backend_name(id, module_id, *, catenate=True, aspect=None):
     if aspect is None:
         aspect = 'table'
-    if aspect != 'table' and not re.match(
+    if aspect not in {'table', 'inhview'} and not re.match(
```

Comment:
Do you have an enum for aspects? Would help with documentation of the code too.

Reply:
It wouldn't be a single enum.  Every schema object has its own set.